extensions/amp-access/0.1/amp-access.js

```diff
@@ -339,7 +339,7 @@ export class AccessService {
 
   /** @private */
   listenToBroadcasts_() {
-    this.viewer_.onBroadcast(message => {
+    this.viewer_.onMessage('broadcast', message => {
```

Comment:
Broadcast shouldn't change - it should stay as it was `onBroadcast`. Broadcast is a special type of messaging, thus we want a special API for it. 

Probably be the easiest to just revert this file and related other.